solutionbox/structured_data/datalab_solutions/structured_data/_package.py

```diff
@@ -220,7 +210,9 @@ def local_train(preprocessed_dir, transforms_config_file, output_dir,
          '--eval_data_paths=%s' % eval_filename,
          '--metadata_path=%s' % metadata_filename,
          '--output_path=%s' % output_dir,
-         '--transforms_config_file=%s' % transforms_config_file,
+         '--schema_file=%s' % schema_file,
```

Comment:
Can preprocessing save the schema file into preprocessed_dir so that we don't have to specify it here?